native/cocos/network/HttpClient-java.cpp

```diff
@@ -683,7 +685,7 @@ void HttpClient::processResponse(HttpResponse *response, char *responseMessage)
     urlConnection.disconnect();
 
     // write data to HttpResponse
-    response->setResponseCode(static_cast<long>(responseCode));
+    response->setResponseCode(static_cast<int32_t>(responseCode));
```

Comment:
Why cast it to int32_t ?

```c++
    inline void setResponseCode(long value) {
        _responseCode = value;
    }
```

Reply:
change to //NOLINT to passby `long` type clang-tidy